src/index.ts

```diff
@@ -1,44 +1,17 @@
-// tslint:disable:no-console no-var-requires no-any
+// tslint:disable:no-console no-any
 import chalk from 'chalk';
 import * as fs from 'fs';
 import * as glob from 'glob';
 import {Plugin as PostCssPlugin} from 'postcss';
 import {Tapable} from 'tapable';
 import {promisify} from 'util';
 import {Compiler} from 'webpack';
-
-const cssModuleCore = require('css-modules-loader-core');
-const DtsCreator = require('typed-css-modules');
-const FileSystemLoader = require('typed-css-modules/lib/fileSystemLoader').default;
+import cssModuleCore = require('css-modules-loader-core');
+import DtsCreator = require('typed-css-modules');
```

Comment:
Why remove the typing here?

Reply:
as the new `typed-css-modules` version is now written in typescript, it comes with its own typings. therefore there is no more need to define them here scetchyly.